### PR TITLE
Some cleanup in gamma scalp open book

### PR DESCRIPTION
**Pull request by brittcyr**
This is intended to be comment and logging changes and no-op changes only.

### CODE REVIEW

File: src/scalper.ts [hunk 1]
```diff
@@ -1064,15 +1047,14 @@ Spread % ${((whaleAskPrice - whaleBidPrice) / fairValue) * 100}`);
...
-      (amountGamma * 2 + backBidQty + backAskQty) * fairValue,
-    );
+    console.log(`${this.symbol} Gamma Spread % ${((priceAsk - priceBid) / fairValue) * 100} \
+Liquidity $ ${(amountGamma * 2 + backBidQty + backAskQty) * fairValue}`);
+
+    // At the base level only, wait for the scalp to fill.
     if (gammaScalpCount === 1) {
       await waitForFill((_) => gammaScalpCount === gammaCycles, scalperWindowSec);
+
+      // TODO: Remove this logging unless necessary. It is not showing new actionable info.
```
Comment (DonDuala) [inline src/scalper.ts:1057]: We should try to record scalps per run potentially but so long as we have good pnl accounting which we dont yet can remove. Useful to have a log when the gamma run has completed still

**Review by DonDuala — APPROVED** [review]

File: src/scalper.ts [hunk 2]
```diff
@@ -797,21 +799,21 @@ Spot Δ: ${spotDelta} Offset Δ ${this.deltaOffset} Fair Value: ${fairValue}`,
...
       gammaIds,
       (message: SerumVialTradeMessage) => {
-        if (message.makerClientId === backBidID.toString()) {
-          console.log(this.symbol, 'Back Bid Fill!!!', tradeMessageToString(message));
-          return;
-        } if (message.makerClientId === backAskID.toString()) {
-          console.log(this.symbol, 'Back Ask Fill!!!', tradeMessageToString(message));
+        if (message.makerClientId === backBidID.toString()
+        || message.makerClientId === backAskID.toString()) {
+          console.log(this.symbol, 'Back Fill!!!', tradeMessageToString(message));
```
[the remark this answers is not in the document]
Comment (brittcyr) [reply src/scalper.ts:804]: It includes the IDs so we should know side it was

File: src/scalper.ts (hunk 1 above)
Comment (brittcyr) [reply src/scalper.ts:1057]: Logging should not be for accounting. We can have separate accounting scripts. Logs are just there for debugging and streamlined to critical messages for fixing an outage.
Can add something in the give up of wait for fill if needed

File: src/scalper.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (DonDuala) [reply src/scalper.ts:804]: Would prefer to leave it explicitly in then

[the remark this answers is not in the document]
Comment (DonDuala) [reply src/scalper.ts:804]: Matching by eye will be annoying

**Pull request merged**